sdks/java/maven-archetypes/examples/src/test/resources/projects/basic/archetype.properties

```diff
@@ -15,7 +15,7 @@
 #    limitations under the License.
 #
 package=it.pkg
-version=0.1-SNAPSHOT
+version=0.1
```

Comment:
[[BEAM-331] Archetype project version shouldn't be coupled to Beam version](https://issues.apache.org/jira/browse/BEAM-331)